lib/verify.js

```diff
@@ -4,13 +4,17 @@ const getError = require('./get-error');
 const resolveConfig = require('./resolve-config');
 
 module.exports = pluginConfig => {
-  const {changelogFile} = resolveConfig(pluginConfig);
+  const {changelogFile, changelogTitle} = resolveConfig(pluginConfig);
   const errors = [];
 
   if (!isUndefined(changelogFile) && !(isString(changelogFile) && changelogFile.trim())) {
     errors.push(getError('EINVALIDCHANGELOGFILE', {changelogFile}));
   }
 
+  if (changelogTitle && !(isString(changelogTitle) && changelogTitle.trim())) {
```

Comment:
Can write this `if` similarly to the one above, using `isUndefined`? I don't remember exactly but I think I encountered a case in which relying on JS falsy conversion created an issue.
```js
!isUndefined(changelogTitle) && !(isString(changelogTitle) && changelogTitle.trim())
```